frontend/src/pages/AccountReceive.vue

```diff
@@ -135,7 +135,8 @@ function useScan() {
       const isFormValid = await settingsFormRef.value?.validate(true);
```

Comment:
@alexkeating this is working great! Nice work.

One downside I noticed testing it is I don't think we have a way to get to the scan settings until *after* the scanning is complete. (Am I wrong about that?). If you're trying to use block restrictions to save time it doesn't really do much good.

What do you think of adding something like what I have mocked up below to the fetching/scanning screens, in advanced mode only. The sentence shown is the same one as shown above the table, just in a different tense.

![image](https://github.com/ScopeLift/umbra-protocol/assets/1479802/7bdf8491-e767-42d4-b509-809e69767cd3)

![image](https://github.com/ScopeLift/umbra-protocol/assets/1479802/fe057b5c-d484-416f-b26f-3e1d3b0f328f)

If this is easy, we can tack it on to this PR. If it's non-trivial for any reason, let's ship this PR and move this into its own issue.

Reply:
https://github.com/ScopeLift/umbra-protocol/issues/522